tools/bridge/docker/docker-compose-nodes-development.yml

```diff
@@ -0,0 +1,41 @@
+version: '2.4'
+services:
+
+  node_foreign:
+    image: parity/parity:v2.5.5-stable
+    command:
+      - "--light"
+      - "--no-download"
+      - "--auto-update=none"
+      - "--chain=ropsten"
+      - "--base-path=/data"
+      - "--no-hardware-wallets"
+      - "--jsonrpc-interface=all"
+      - "--jsonrpc-apis=safe"
+      - "--jsonrpc-hosts=all"
+      - "--jsonrpc-cors=all"
+      - "--no-ipc"
+      - "--no-secretstore"
+      - "--no-color"
+    volumes: ['./data/ropsten:/data']
+    restart: unless-stopped
+    mem_limit: 2G
+    mem_reservation: 16M
+    stop_grace_period: 1m
```

Comment:
Question: I think Parity's own default is 3min. Do we want to quit earlier then there are problems/

Reply:
Changed.